app.R

```diff
@@ -0,0 +1,1382 @@
+               href = "https://pennstate.qualtrics.com/jfe/form/SV_7TLIkFtJEJ7fEPz?appName=Log_Transformations"
+        )
+      ), 
+      tags$li(class='dropdown', 
+              tags$a(href="https://shinyapps.science.psu.edu/", 
+                     icon('home', lib='font-awesome')))), 
+    dashboardSidebar(
+      width = 250, 
+      sidebarMenu(
+        id = "pages", 
+        menuItem("Overview", tabName = "overview", icon = icon("tachometer-alt")), 
+        menuItem("Explore", tabName = "explore", icon = icon("wpexplorer")), 
+        menuItem("References", tabName = "References", icon = icon("leanpub"))
+      ), 
+      tags$div(
+        class = "sidebar-logo", 
+        boastUtils::psu_eberly_logo("reversed")
+      )
+    ), 
+    dashboardBody(
+      tabItems(
+      tabItem(tabName = "overview", 
+              h1("Log Transformation"), 
+              p("The goals of this app are to know when to use log 
+              transformations to linearize data, how to analyze scenarios
+                where a Log Transform is needed and when it is not."), 
+              p("There are three different datasets, which are about Animals, 
+                Earthquakes, and Countries."), 
+              br(), 
+              h2("Instructions"), 
+              tags$ol(
+                tags$li("In the application, you will see a number of 
+                        different data sets you can go through."),  
+                tags$li("Go through each data set and look through 
+                the different variables to check whether
+                        a log transformation would help."),  
+                tags$li("Use the 'Show Histogram' button to see each 
+                        individual variable more clearly."),  
+                tags$li("You can also examine log transforms on 
+                        variables in your own data set!")
+              ), 
+              div(
+                style = "text-align: center", 
+                bsButton(
+                  inputId = "go1", 
+                  label = "GO!", 
+                  size = "large", 
+                  icon = icon("bolt"), 
+                )
+              ), 
+              br(), 
+              br(), 
+              h2("Acknowledgement"), 
+              p("This app was coded and developed by Alex Chen. The was further
+              updated by Daehoon Gwak in November 2020.", 
+                br(), 
+                br(), 
+                br(), 
+                div(class = "updated", "Last Update: 12/10/2020 by DG")
+              )
+      ), 
+    tabItem(tabName = "explore", 
+            h2("Log Transformation Task"), 
+              fluidRow(
+                  column(
+                    width = 3,
+                    selectInput(
+                      inputId = "inputs",
+                      label = "Select Data Set",
+                      choices = c('Animals', 'Earthquakes', "Countries")
+                    ) 
```

Comment:
Whenever I select the Earthquakes data set it says magnitude not found. I thought this was fixed in the meeting so maybe you forgot to push your code?